src/main/java/org/elasticsearch/search/aggregations/metrics/stats/extended/InternalExtendedStats.java

```diff
@@ -104,6 +124,33 @@ public double getStdDeviation() {
     }
 
     @Override
+    public double getStdDeviationBound(Bounds bound) {
+        if (bound.equals(Bounds.UPPER)) {
+            return getAvg() + (getStdDeviation() * sigma);
+        } else {
+            return getAvg() - (getStdDeviation() * sigma);
+        }
+    }
+
+    @Override
+    public double getStdError() {
+        return getStdDeviation() / Math.sqrt(count);
+    }
+
+    @Override
+    public double getConfidenceBound(Bounds bound) {
+        if (confidenceInterval == -1) {
+            return Double.NaN;
```

Comment:
Hm, so this is how I'm signifying that the user did not ask for a confidence boundary.  In the REST API, we just don't return that part of the response.  But in the Java API, you can always `getConfidenceBound()`.

Since the confidence intervals carry certain semantics depending on your data, we didn't want to accidentally mislead a developer who doesn't know how to use it.  

Would an exception saying "You didnt ask for an interval" be more or less friendly than a NaN?